server/ngwmn/services/ngwmn.py

```diff
@@ -40,9 +41,21 @@ def get_iddata(request, agency_cd, location_id, service_root=SERVICE_ROOT):
         app.logger.error(msg, resp.status_code, resp.url, resp.reason)
         raise ServiceException()
 
+    app.logger.debug('Got %s response from %s', resp.status_code, resp.url)
     return parse_xml(resp.content)
 
 
+def _(parent, tag):
```

Comment:
Why just '_' as the name. Why not _find?

Reply:
I did so it's concise, but `_find` is more clear 👍